packages/keyring-controller/src/KeyringController.ts

```diff
@@ -1028,7 +1082,7 @@ export class KeyringController extends BaseController<
       }
       const newKeyring = (await this.#newKeyring(KeyringTypes.simple, [
         privateKey,
-      ])) as EthKeyring<Json>;
+      ], { id: ulid(), name: '' })) as EthKeyring<Json>;
```

Comment:
I looks like we are always passsing the same object to `this.#newKeyring`:

```typescript
{
  id: ulid(),
  name: '',
}
```

Have you considered moving this directly to `#newKeyring`, removing the need for an additional method argument?